back/main.py

```diff
@@ -3,12 +3,19 @@
 from scripts.utils.logger_manager import LoggerManager
 from scripts.workflow.workflow_manager import WorkflowManager
 
+from back.scripts.utils.config import project_config
 from back.scripts.workflow.data_warehouse import DataWarehouseWorkflow
 
 if __name__ == "__main__":
     # Parse arguments, load config and configure logger
     args = ArgumentParser.parse_args("Gestionnaire du projet LocalOuvert")
+
+    # Load config file
     config = ConfigManager.load_config(args.filename)
+
+    # Load project configuration instance
```

Comment:
I think those comments do not bring more information.

Reply:
Yeah but it's a bit confusing without it :
```py
config = ConfigManager.load_config(args.filename)
project_config.load(config)
```
What's the difference between charging the configuration twice?